XenAdmin/Wizards/ImportWizard/ImportSourcePage.cs

```diff
@@ -244,9 +242,7 @@ private bool GetDiskCapacityXva(out string error)
 				FileInfo info = new FileInfo(m_textBoxFile.Text);
 				ImageLength = info.Length > 0 ? (ulong)info.Length : 0;
 
-				DiskCapacity = IsXvaVersion1
-				               	? GetTotalSizeFromXmlGeneva() //Geneva style
-				               	: GetTotalSizeFromXmlXva(GetXmlStringFromTarXVA()); //xva style
+				DiskCapacity = GetTotalSizeFromXmlXva(GetXmlStringFromTarXVA()); //xva style

```

Comment:
The function GetTotalSizeFromXmlGeneva() is (I assume) no longer called and can be removed.

Also applyVersionOneFiles() from ImportVmAction.cs.